src/main/java/mate/academy/springboot/swagger/util/SortParser.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.springboot.swagger.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.data.domain.Sort;
+
+public class SortParser {
+    // products?count=20&page=1&sortBy=price:ASC;title:ASC
+    public static Sort parse(String sortBy) {
+        List<Sort.Order> orders = new ArrayList<>();
+        if (sortBy.contains(":")) {
+            String[] sortedFields = sortBy.split(";");
+            for (String field : sortedFields) {
+                Sort.Order order;
+                if (field.contains(":")) {
+                    String[] fieldsAndDirections = field.split(":");
+                    order = new Sort.Order(Sort.Direction.valueOf(fieldsAndDirections[1]),
+                            fieldsAndDirections[0]);
```

Comment:
let's introduce constants with meaningful names for indexes 1 and 0
same for ":" and ";"